src/GovUk.Education.ExploreEducationStatistics.Content.Model/Database/ApplicationDbContext.cs

```diff
@@ -1544,9 +1544,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                                         TimePeriod = new TimePeriod
                                         {
                                             StartYear = "2012",
-                                            StartCode = TimeIdentifier.AcademicYear,
+                                            StartCode = TimeIdentifier.SixHalfTerms,
```

Comment:
Subject Id=1 TimeIdentifiers are AcademicYear not SixHalfTerms. This seems to be reverting an earlier change.

Reply:
Fixed after updating due to data